components/Form/EditInfo.jsx

```diff
@@ -0,0 +1,30 @@
+import { View, Text, TouchableOpacity } from "react-native";
+import FormField from "../Form/FormField.jsx";
+
+export default function EditInfo(props) {
+    return (
+        <View className="w-10/12 flex flex-col gap-2">
+            <Text className="text-2xl font-extrabold text-center mb-1">{props.title}</Text>
+            <FormField
+                value={props.variableToEdit}
+                onChangeText={(text) => props.onChangeFunction(text)}
+            />
+            <View className="flex flex-row justify-around">
+                <TouchableOpacity
+                    className="bg-green-200 border-2 border-black rounded-3xl p-2 flex items-center justify-center w-5/12"
+                    onPress={() => {props.onSubmitFunction()}}
+                >
+                    <Text className="text-black font-bold">Submit</Text>
+                </TouchableOpacity>
+                <TouchableOpacity
```

Comment:
can we use the form button component for these? might need to allow for background/width styling in the component but could be more consistent

Reply:
Made SmallFormButton